Framework/Algorithms/test/Stitch1DManyTest.h

```diff
@@ -388,6 +388,7 @@ class Stitch1DManyTest : public CxxTest::TestSuite {
     alg.setProperty("StartOverlaps", "0.8, 1.6");
     alg.setProperty("EndOverlaps", "1.1, 1.8");
     alg.setProperty("OutputWorkspace", "outws");
+    alg.setProperty("IndexOfReference", "0");
```

Comment:
If 0 is the default, then you shouldn't need to specify here.
In fact, it makes a good test: if you don't specify it, does the test still pass?
This ensures backwards compatibility.

Reply:
It absolutely passes, I'll remove that.